### ISSUE TITLE
Apple pay and Google pay for web 

### ISSUE THREAD

**Issue by manandhar-aarya** [flutter-stripe/flutter_stripe#1851]
**Describe the bug**
In Flutter web, Stripe.instance.isPlatformPaySupportedListenable.value always returns false. Even though the devices has Apple pay enabled and I can pay using JS interop bridge with Stripe.js. 

I see on the changelog for Flutter_stripe 10.1.0 "#1122 Add google pay and apple pay for web" but I am not able to use it.  I checked on the example project but on the web version it says "Not available on this device". Could you please add a sample code for the web version of Apple and Google pay? 

**Expected behavior**
Apple pay and Google pay button should be visible in appropriate device and it should be possible to pay with them. 

**Smartphone / tablet**
 - Device: [e.g. MacBook Pro, iPhone 13, Samsung Galaxy A13]
 - OS: [e.g. iOS 17.5.1, Android 13]
 - Package version: [flutter_stripe: 10.2.0, flutter_stripe_web: 5.2.0]
 - Flutter version [ 3.22.2]

**Comment by remonh87** [comment]
Platform pay is at the moment only supported for mobile. But you can use apple and google payments through the stripe paymentelement for the web that is part of the flutter_stripe_web library.

**Comment by mattostanik** [comment]
I attempted to use flutter_stripe_web to implement Apple Pay for Flutter Web, but I was unable to figure out how to get it to work. Instead I ended up using a Stripe Payment Request Button with Stripe.js in an HTML file, then I embedded the HTML file in my Flutter screen using an HtmlElementView.

I wrote a longer explanation that shows all of the steps to do this as an answer to this question:
https://stackoverflow.com/questions/76971497/implement-payment-element-stripe-with-flutter-web

Sharing it here in case anyone else (similar to me) finds this issue and discussion while trying to figure out how to use Apple Pay or platform pay in Flutter web.
